compiler-cli/src/new.rs

```diff
@@ -121,7 +121,7 @@ pub fn main() {{
             self.root.join(".gitignore"),
             "*.beam
 *.ez
-build
+build/
```

Comment:
It needs to match only at the root, so the `/` should be at the start

Reply:
Sorry, fixed!